datasets/aster/tests/test_tasks.py

```diff
@@ -0,0 +1,14 @@
+# Note you will have to run this with `python -m pytest` from the datasets/aster directory
+
+from pathlib import Path
+
+import aster
+import geopandas
+import orjson
+
+
+def test_update_geometries_from_dataframe() -> None:
+    path = Path(__file__).parent / "data-files" / "aster-l1t-subset.parquet"
+    dataframe = geopandas.read_parquet(path)
+    item = next(aster.update_geometries_from_dataframe(dataframe))
```

Comment:
This maybe works, but may not be in the spirit of the original test:

```python
from pathlib import Path

import aster
import geopandas
import orjson

from pctasks.task.context import TaskContext
from pctasks.core.storage import StorageFactory

def test_update_geometries_from_dataframe() -> None:
    path = Path(__file__).parent / "data-files" / "aster-l1t-subset.parquet"
    input = aster.CreateChunksInput(asset={"href": str(path), "table:storage_options": {}, "partition-number": 0}, dst_uri="dest.txt", chunk_size=10)
    
    result = aster.create_chunks_task.run(input, TaskContext(storage_factory=StorageFactory(), run_id="abc"))
    assert result  # do something useful here?
```

I can't decide whether our tests should be asserting anything about the actual output data written here. It feels like this task should be very simple (geoparquet of STAC items to (chunked) ndjson of STAC items?) but you're having to hack around some stuff.

Reply:
Actually, that doesn't quite cover the spirit of the original test.

IMO, the thing we should really be checking is the output of `sign_and_update` on the output of a single item. Can you make sure that's covered by the tests somewhere?

All the other stuff is (I think) plumbing that should arguably be handled by some core functionality in pctasks. I can take that on as a followup task.

Reply:
A simple manual test looks to be OK though.

```python
In [26]: item = pystac.read_dict(json.loads(p.read_text().split("\n")[0]))

In [27]: result = aster.sign_and_update(item, simplify_tolerance=0.001)
```

So I think just a small test asserting that stuff works for an item will be sufficient for me.

Reply:
Yeah, since that test wasn't being hit by CI, I forgot about it and it bit-rotted. I've now updated it to be pretty close to what you have for you manual test.